gremlin-python/pom.xml

```diff
@@ -311,7 +311,7 @@ limitations under the License.
                                             <fileset dir="src/main/jython"/>
                                         </copy>
                                         <!-- print versions of python and virtualenv -->
-                                        <exec dir="${project.build.directory}/python2" executable="python2"
+                                        <exec dir="${project.build.directory}/python2" executable="python"
```

Comment:
I think I'd like to keep the same assumptions we've always had about local dev environments as we're deep into 3.4.x at this point and python2 is gone in 3.5.0. Do you mind reverting this change and doing a alias or something in your environment locally to accommodate this configuration?